hw/ip/prim/dv/prim_alert/data/prim_alert_cover.cfg

```diff
@@ -0,0 +1,10 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
++module prim_alert_sender
```

Comment:
Seems like we can now safely blackbox `prim_alert|esc_sender|receiver` from our common cover.cfg.

Reply:
+1